src/app/policies/views/PolicyListView.vue

```diff
@@ -117,7 +117,7 @@
                             </KSelect>
 
                             <DocumentationLink
-                              :href="t('policies.href.docs', {'name': selected.name})"
+                              :href="t('policies.href.docs', { name: selected.name.toLowerCase() })"
```

Comment:
In relation to the above comment. You can see here how we are now using `t` for hrefs/URLs.

Also, I'm pretty sure @lahabana wanted the URLs here to use the exact unmodified name (i.e. without the lowercasing). I might have misunderstood or things may have changed since then, but it's probably a good idea to double check with him.

Reply:
This breaks the URLs though. The redirects on the docs website don’t work when using the plain policy type names.

Reply:
Yes I know, I asked that also when this was done. I'd understood they were going to be fixed on the docs site. I might be completely wrong but I think we should at least check before changing this as I think it was done like this on purpose.

Reply:
I have checked this (hence my addition of this fix) and it’s not working like that.

Navigating to a docs URL without the lowercasing which also relies on a redirect will currently 404 because the redirect won’t match against the non-lowercased value.

Reply:
>I have checked this (hence my addition of this fix) and it’s not working like that.

ok well as I said that was the case when @lahabana added this and we discussed it at the time, so I'll have to leave it to him as I just don't know either way. Maybe since then it was agreed to lowercase these?

Reply:
There was no agreement of that kind, but it was observed in a couple of places that the documentation URLs are broken and that’s one of the issues. If we decide to make the redirects more robust to work without the lowercasing, we can easily remove this here.

Until then, I much rather have the documentation links working.